src/constants/urlConstants.ts

```diff
@@ -0,0 +1,5 @@
+const BUSINESS_URL = '/v3/businesses';
+
+const SEARCH_BUSINESS = BUSINESS_URL + '/search';
+
+export {SEARCH_BUSINESS};
```

Comment:
Do not export as individual keys. Instead export as ApiEndpoints. Also, move this to within api folder